tests/unit/test_metrics.py

```diff
@@ -978,13 +1128,19 @@ def test_compute_float_block_size(self) -> None:
         # raised from the range function should provide all information needed for users
         # to identify the issue
         with self.assertRaises(TypeError) as error_raised:
-            metric.compute(x=self.x, y=self.y, block_size=2.0)
+            metric.compute(
+                reference_data=self.reference_data,
+                comparison_data=self.comparison_data,
+                block_size=2.0,
+            )
         self.assertEqual(
             error_raised.exception.args[0],
             "block_size must be a positive integer",
         )
 
 
+# pylint: enable=duplicate-code
+# pylint: enable=too-many-locals
```

Comment:
These two seem redundant - they don't have a corresponding `disable` in this file

Reply:
Agree, fixed